dev/node-networking.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+echo '
+network:
+    ethernets:
+        eth0: 
+            dhcp4: true
+            dhcp4-overrides:
+                use-dns: false
+                use-routes: false 
+' > /etc/netplan/01-netcfg.yaml
+echo "
+    network:
+        ethernets:
+            eth1:
+                dhcp4: false
+                addresses: [10.10.0.$1/24]
+                routes:
+                -   to: default
+                    via: 10.10.0.2 
+                    metric: 0
+                nameservers:
+                    addresses: [1.1.1.1, 1.0.0.1]
+        version: 2
+" > /etc/netplan/50-privatenetwork.yaml
```

Comment:
Well, because it's only a dev cluster specific config and probably won't be changed much, I don't think it's very necessary to seperate this, it also has the benefit of node network configuration being in one file

Reply:
Ia there any possibility that the values in these configs might change? I got a slight ick seeing those values there. Maybe I'm being way too perfectionist, I'll admit. Your thoughts?

Reply:
This comment mostly referentes addresses, boolean values are whatever

Reply:
> This comment mostly references addresses, boolean values are whatever

Addresses are okay to hardcode, since they are static, and only relevant to the dev cluster.

Reply:
I'll give my 2 cents, but I'm okay if it stays as it is: I think we should have a synced folder (shared between the host and the guests). This is because to set up the cluster, it would be easier to have all of the commands in bash scripts and just run them in the guest VMs. And if we had synced folders, the setup script would be just three lines (copy the netplans, apply them, reboot) and, if someone needed to change them for whatever reason, I think it would be easier to do so and understand what's happening. I also think that these are pretty minor things and thus I'm okay with it staying as it is.